### ISSUE TITLE
feature request - show miner version in webpanel

### ISSUE THREAD

**Issue by hidden-69** [UnamSanctam/UnamWebPanel#111]
So I know which miners I have to update

**Comment by UnamSanctam** [comment]
Hmm, it would mean I'd have to update the miners themselves on each update but it's possible to do.

**Comment by hidden-69** [comment]
Would be better to only update the miner level when something changes in the miner, else everyone is updating their miner when nothing has actually changed..

**Comment by hidden-69** [comment]
Could also let users define their own "report" field in the miner build that get's reported to the web panel. Users can create their own versions this way.

**Comment by UnamSanctam** [comment]
It's possible to do, it's just a bit more work since I'll have to either change the version myself in the binaries (recompile miners for each version) or I'd have to add it into the command line so that the builder tells the version, the second one is easier but will take up more of the limited space.

**Comment by hidden-69** [comment]
How about a custom field? This way I can also define a version number for different builds

**Comment by UnamSanctam** [comment]
Yes if I do add one then it will most likely told to the miners by the builder (via a command line option), which would also allow it to be customized.

**Comment by hidden-69** [comment]
Cool! Number of shares submitted by miner would be nice as well, a way to filter on the miners I need to investigate for not mining ever or just temporary.
